### DISCUSSION TITLE
阿里云webdav插件安装显示与架构不匹配

### DISCUSSION THREAD

**Discussion by lpark365** [messense/aliyundrive-webdav#662] [Q&A]
路由架构：arch arm_cortex-a7 10，路由器：GL-AXT1800

固件版本：固件版本	OpenWrt 21.02-SNAPSHOT r16399+157-c67509efd7 / LuCI openwrt-22.03 branch git-21.284.67084-e4d24f0

按照大佬的库内下载了很多安装都失败，提示与架构不匹配，请大佬协助。

**Comment by SunWeiKeS** [comment]
兄弟我和你一样，请问你解决了吗

**Comment by lpark365** [comment]
已经解决，阿里云的插件需要用之前的版本







&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "messense/aliyundrive-webdav"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年1月10日(星期二) 晚上10:55
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [messense/aliyundrive-webdav] 阿里云webdav插件安装显示与架构不匹配 (Discussion #662)





 
兄弟我和你一样，请问你解决了吗
